docs/user-guide-policy-configure-ovs-bridge.md

```diff
@@ -0,0 +1,86 @@
+# Tutorial: Create a Open vSwitch Bridge and Connect it to a Node Interface
+
+Use Node Network Configuration Policy to configure a new ovs bridge `br1` connected
+to node interface `eth1`.
+
+## Requirements
+
+Before we start, please make sure that you have your Kubernetes/OpenShift
+cluster ready. In order to do that, you can follow the guides of deployment on
+[local cluster](deployment-local-cluster.md) or your
+[arbitrary cluster](deployment-arbitrary-cluster.md).
+
+You must also be sure that [OpenVSwitch](https://www.openvswitch.org/) and [NetworkManager OpenVSwitch plugin](https://developer.gnome.org/NetworkManager/stable/nm-openvswitch.html) are installed in the nodes.
+
+Please also note that on OpenShift `openvswitch` is already installed as a daemon, so only `nm-openvswitch` is required.
```

Comment:
This is true only for OpenShift. On Kubernetes, we don't know.

Reply:
Isn't what the comment says?

Reply:
Tired me. Nevermind.